### ISSUE TITLE
The option with multiples files is not working

### ISSUE THREAD

**Issue by froucher** [felixzapata/gulp-stubby-server#9]
`options.files` just works with the first element of the array.

For example
```js
var options = {
    stubs: 8000,
    mute: false,
    location: 'localhost',
    relativeFilesPath: false,
    files: [
        '/mocks/**/*-config.yaml',
        '/mocks/**/*-config.json'
    ]
};
 
gulp.task('stubby', function(cb) {
    return stubby(options, cb);
});
```

We are trying to work with `'/mocks/**/*-config.yaml'` and `'/mocks/**/*-config.json'`, but just the first element is working.

**Comment by felixzapata** [comment]
hi, at the moment you can use like this way:

```javascript
files: [
        '/mocks/**/*-config.{json,yaml}'
    ]
```
However, I will review it to fix this issue and admit more than one entry.

**Comment by froucher** [comment]
Thx!

**Comment by felixzapata** [comment]
@froucher fixed in [release 0.2.0](https://github.com/felixzapata/gulp-stubby-server/releases/tag/v0.2.0)
